fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/FhirResourceTableGroup.java

```diff
@@ -397,6 +400,14 @@ public void addDateValues(List<IDatabaseObject> group, String prefix) {
                 .setTablespace(fhirTablespace)
                 .addPrivileges(resourceTablePrivileges)
                 .enableAccessControl(this.sessionVariable)
+                .addPostStep(priorVersion -> {
+                    List<IDatabaseStatement> statements = new ArrayList<>();
+                    if (priorVersion == 1) {
+                        statements.add(new DropColumn(schemaName, tableName, "DATE_VALUE"));
```

Comment:
I don't think we should drop this column.  I think we leave it for a minimum number of versions.

Reply:
If we decide to keep it we should use a constant for DATE_VALUE_DROPPED_COLUMN something to clearly indicate don't use it.

Reply:
We decided not to add migration logic which drops the column, but did we remove it from v2 of the table.  I added the following comment to try making it clear:
```
                        // Note: version 1 of this table had a DATE_VALUE column.
                        // We chose not to DROP it here, but it is not used going forward.
```

Reply:
Now that I added the DROP back in, I used this suggestion to use a constant named `DATE_VALUE_DROPPED_COLUMN`